package.json

```diff
@@ -69,6 +69,14 @@
       "es2015"
     ]
   },
+  "nyc": {
+    "extension": [
+      ".raw"
```

Comment:
Ah, you'll probably need to put both raw and js here then because you're overriding the default.